tests/conftest.py

```diff
@@ -191,6 +194,16 @@ def rand_one_dut_hostname(request):
         dut_hostnames = random.sample(dut_hostnames, 1)
     return dut_hostnames[0]
 
+@pytest.fixture(scope="module")
+def rand_one_frontend_dut_hostname(request):
+    """
+    """
+    frontend_dut_hostnames = generate_params_frontend_hostname(request)
+    if len(frontend_dut_hostnames) > 1:
+        dut_hostnames = random.sample(frontend_dut_hostnames, 1)
+    return dut_hostnames[0]
```

Comment:
`dut_hostnames` would be undefined if `len(frontend_dut_hostnames) <= 1`. Exception could be triggered.

Reply:
Thanks for pointing this.  Resolved in latest commit